packages/orchestrator/lib/clients/client.ts

```diff
@@ -40,11 +40,20 @@ export class OrchestratorClient {
     private routeFetch<E extends Endpoint<any>>(
         route: Route<E>,
         config?: {
-            timeoutMs: number;
+            timeoutMs?: number;
+            retryConfig?: RetryConfig<E['Reply']>;
         }
     ): (props: { query?: E['Querystring']; body?: E['Body']; params?: E['Params'] }) => Promise<E['Reply']> {
         return (props) => {
-            return retryWithBackoff(() => routeFetch(this.baseUrl, route, config)(props), { retry: httpRetryStrategy, numOfAttempts: 3, maxDelay: 50 });
```

Comment:
had to refactor `retry.ts` because it only handles function that throw error. The orchestrator typed API isn't throwing error but return them as values